cli/plugins/config-per-target/README.md

```diff
@@ -0,0 +1,29 @@
+# buildbuddy-io/plugins:config-per-target
+
+The config-per-target plugin allows you to specify arguments to pass to
+executables run by bazel on a per-target basis. The arguments will be prepended
+to the options specified on the command line, as this will sometimes allows
+options specified on the command line to override those specified as default
+options by this plugin (dependent on the implementation of the application).
+
+## Installation
+
+Install this plugin with:
+
+```
+bb install buildbuddy-io/plugins:config-per-target
+```
+
+## Configuration
+
+Create a `config.json` file in the plugin directory. The JSON should specify a
```

Comment:
The plugin directory will be under `~/.cache` when installed like `bb install buildbuddy-io/plugins:config-per-target` on Linux. When the user clears their cache (e.g. to reclaim disk space) we don't want to blow away this config.

What do you think about putting this in `$BUILD_WORKSPACE_DIRECTORY/.config-per-target.json` ? That way it's also clear that the config only applies to the current Bazel workspace. (The downside is that it requires modifying .gitignore)

We could also put it in `$USER_CONFIG_DIR/config-per-target/config.json` but it should probably be namespaced by repo somehow, e.g. 

```
{
  "/path/to/workspace1": {
    "//target1": ["..."],
  }
}
```

Reply:
Works for me! Also, this is the kind of file I just put in `.git/info/exclude`, honestly. I actually already have :P